.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
 script:
     - npm test
     - env
+    - eslint
```

Comment:
Let's make a `"lint"` script in package.json (like `"test"`) that runs `eslint`.